modules/codegen-plugin/src/sbt-test/codegen-plugin/custom-settings/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.0
+sbt.version=1.5.5
```

Comment:
I had to do this to run any scripted tests, otherwise it said this:

```
[error] [error] [launcher] error during sbt launcher: java.lang.UnsatisfiedLinkError: Can't load library: /Users/velvetbaldmime/Library/Caches/JNA/temp/jna4456582555411290288.tmp
[error] x codegen-plugin/defaults
sbt.internal.scripted.TestException: {line -1}  Reload for scripted batch execution failed.
  | => cat sbt.scriptedtest.ScriptedTests.$anonfun$runBatchedTests$5(ScriptedTests.scala:237)
	at sbt.scriptedtest.ScriptedTests.$anonfun$runBatchedTests$5$adapted(ScriptedTests.scala:231)
	at sbt.scriptedtest.ScriptedTests.$anonfun$commonRunTest$6(ScriptedTests.scala:325)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at sbt.scriptedtest.ScriptedTests.commonRunTest(ScriptedTests.scala:324)
	at sbt.scriptedtest.ScriptedTests.$anonfun$runBatchedTests$3(ScriptedTests.scala:241)
	at sbt.scriptedtest.ScriptedTests.runOrHandleDisabled(ScriptedTests.scala:275)
	at sbt.scriptedtest.ScriptedTests.$anonfun$runBatchedTests$1(ScriptedTests.scala:245)
```

Which I think is a known JNA issue on M1 macs in older versions of SBT.

Which also makes me think none of y'all are running scripted tests locally 🤠

Reply:
Considering the SBT plugin is just a small wrapper over the logic held by the `codegen` module, and that we test that via bootstrapping, I don't really care about the scripted tests that much 😄

Reply:
but thanks anyway